packages/react/src/components/Button/Button.tsx

```diff
@@ -21,66 +18,55 @@ export type ButtonProps = {
    */
   size?: 'sm' | 'md' | 'lg';
   /**
-   * If `Button` should fill full width of its container
+   * If `Button` should fill width of its container
    * @default false
    */
-  fullWidth?: boolean;
-  /** Toggle icon only styling, pass icon as children
+  fill?: boolean;
+  /** Toggle loading state
    * @default false
    */
-  icon?: boolean;
+  loading?: boolean;
   /**
    * Change the default rendered element for the one passed as a child, merging their props and behavior.
    * @default false
    */
   asChild?: boolean;
-} & ButtonHTMLAttributes<HTMLButtonElement>;
+} & React.ButtonHTMLAttributes<HTMLButtonElement>;
```

Comment:
Why use `React.*` for types? Can't we keep using imports? I feel it's more readable without the `React.*`

Reply:
Det er nok smak og behag, men jeg personlig synes det er svært forvirrende når `ButtonHTMLAttributes ` og `HTMLButtonElement` er så lik i from - hvilken av dem kommer fra Typescript DOM vs. React